fs-one-concepts.md

```diff
@@ -3,19 +3,29 @@
 Try your best to answer each question on your own before looking up the answer online. Once you're done writing your first answer, you can google the question and write the best answer you find.
 
 #### 1. What is Enzyme and what are some of the methods that it provides?
-
+    Testing in react by simulating user interactions and checking the outcome.
+    Some methods that it provides are mount, find, and simulate.
 #### 2. What is the difference between dynamic and a static routes?
-
+    Static routes are hard coded and dynamic routes are generated automaticly based on the logic put in place.
 #### 3. What is a JSON API?
-
+    Its the language the backend and the frontend communicate with. Holding data in the JSON format which is like javascript objects.
 #### 4. What is a migration and why would you use one?
-
+    A migration is what up dates the database based on what you put in the created migration.
 #### 5. Explain how to set up a route in React.
-
+    You would need to go to the roots file and set up a root by first declearing what http action you would use such as get, post, and destroy
+    then you need to link the controller to its methods   controller => main  method => index    main#index
```

Comment:
This is actually a great example of how routes are set up in Rails. How does it look in a React component?